client/extensions/woocommerce/app/settings/payments/payment-method-edit-form-toggle.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from 'components/forms/form-toggle';
+
+const PaymentMethodEditFormToggle = ( { checked, name, onChange } ) => {
```

Comment:
Should we do this when another component needs to use it?  Can I do this in another PR as several other PR's depend on this one?